### DISCUSSION TITLE
Mention App Store Connect API in Continuous Integration docs

### DISCUSSION THREAD

**Discussion by sonicdoe** [fastlane/fastlane#18123] [Ideas] [answered]
The [Continuous Integration](https://docs.fastlane.tools/best-practices/continuous-integration/) page is one of the first search results for “[fastlane authentication](https://duckduckgo.com/?q=fastlane+authentication)”. However, the [App Store Connect API](https://docs.fastlane.tools/app-store-connect-api/), which actually is the recommended way of authentication, isn’t mentioned at all on this page.

I’d suggest to mention and link to it under the “Authentication with Apple services” heading.

**Comment by max-ott** [comment, marked as answer]
Fastlane is open source. Please open a pull request and change the docs in it.

**Reply by sonicdoe** [reply]
I hope my “question” didn’t come across as dismissive. My intention was simply to suggest an improvement to the documentation. I didn’t open an issue because the [New Issue](https://github.com/fastlane/fastlane/issues/new/choose) page directed me to Discussions.

In any case, with https://github.com/fastlane/docs/pull/1032, the page now mentions the App Store Connect API as the recommended method.
